src/components/puck/atoms/heading.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 import { cva, type VariantProps } from "class-variance-authority";
-import { type NumberOrDefault } from "../../editor/NumberOrDefaultField.tsx";
+import { type NumberOrDefault } from "../../editor/NumberOrDefaultField.js";
```

Comment:
Regression back to .js? But I'm assuming it's related to the requirements for components.

Reply:
The starter's tsconfig doesn't allow ts imports -- can update that and keep these as .tsx if we want

Reply:
Nah this is ok